packages/frontend/app/services/current-user.ts

```diff
@@ -68,30 +80,39 @@ export default class CurrentUserService extends Service {
     return toHex(this.publicKey);
   }
 
+  get signingKeyHex(): string {
+    if(!this.publicSigningKey) return '';
+
+    return toHex(this.publicSigningKey);
+  }
+
   get shareUrl(): string {
-    const uri = `${ENV.host}/invite?name=${this.name}&publicKey=${this.uid}`;
+    const uri = `${ENV.host}/invite?name=${this.name}&publicKey=${this.uid}&publicSigningKey=${this.signingKeyHex}`;
```

Comment:
why do we need to have the signing key in the URL?

Reply:
the public key should be enough to identify them

Reply:
Just removed this. No longer storing public signing keys as part of contacts. Will only use them when received with votes.